tests/Google/Service/ResourceTest.php

```diff
@@ -138,12 +138,35 @@ public function testCallServiceDefinedRoot()
     $this->assertEquals("https://sample.example.com/method/path", (string) $request->getUri());
     $this->assertEquals("POST", $request->getMethod());
   }
-
+  
+  /* Some Google Service (Google_Service_Directory_Resource_Channels and Google_Service_Reports_Resource_Channels) use a different servicePath value that should override 
+  the default servicePath value, it's represented by a / before the resource path. All other Services have no / before the path*/
```

Comment:
Can we get this guy to follow coding standards? E.g. wrap at 80 characters and use proper comment blocks:

```
/**
 * Some Google Service (Google_Service_Directory_Resource_Channels and 
 * Google_Service_Reports_Resource_Channels) use a different servicePath value 
 * that should override the default servicePath value, it's represented by a / 
 * before the resource path. All other Services have no / before the path
 */